WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyControlTileViewModel.cs

```diff
@@ -125,12 +136,15 @@ private void Update()
 			var privateAmount = _wallet.Coins.FilterBy(x => x.HdPubKey.AnonymitySet >= privateThreshold).TotalAmount();
 			var normalAmount = _wallet.Coins.FilterBy(x => x.HdPubKey.AnonymitySet < privateThreshold).TotalAmount();
 
+			var queued = _wallet.Coins.FilterBy(x => x.CoinJoinInProgress).TotalAmount().ToDecimal(MoneyUnit.BTC);
```

Comment:
How about renaming this to `queuedDecimalAmount` to be consistent with naming of the other variables.